server/functions.php

```diff
@@ -55,7 +55,6 @@ static function getRender($widget_name, $settings, $ws, $hs) {
 // Image rendering stuff
 
 function renderSVG($id) {
-	header('Content-type: image/svg+xml');
```

Comment:
Good point, I actually planned to investigate a bit more why this causes a problem. My current assumption is: rendering the bitmap formats also calls this function. Setting the header here writes the header 'Content-type: image/svg+xml' into the png/proprietary bitmap output stream. If I don't remove this line the bitmap is not displayed on the home screen of the admin page. Probably the header(..) call should be moved to the place where svg, png and proprietary bitmap are processed.

Reply:
Errr good catch! Yeah this should indeed be moved in that case.
However I do not agree with the statement that header() writes to the bytestream. In PHP, if you already sent the headers, any header() results in an error (headers already sent).
I'm hesitant to merge this line change :D

Reply:
The error is exactly the reason, why I had to remove the line. If the line is present the preview image cannot be rendered, because of the error you mentioned

Reply:
Oh I see, makes sense. I guess my php.ini doesnt enable warnings on the page (you can disable that to the php error log where it belongs). Not nice that users can see whats going on :P
Can we at least move this line to the correct place?